src/WebSocketsServiceProvider.php

```diff
@@ -26,23 +27,30 @@ public function boot()
             __DIR__.'/../config/websockets.php' => base_path('config/websockets.php'),
         ], 'config');
 
-        if (! Schema::hasTable('websockets_statistics_entries')) {
-            $this->publishes([
-                __DIR__.'/../database/migrations/create_websockets_statistics_entries_table.php.stub' => database_path('migrations/'.date('Y_m_d_His', time()).'_create_websockets_statistics_entries_table.php'),
-            ], 'migrations');
-        }
+        try {
+            if (! Schema::hasTable('websockets_statistics_entries')) {
+                $this->publishes([
+                    __DIR__.'/../database/migrations/create_websockets_statistics_entries_table.php.stub' => database_path('migrations/'.date('Y_m_d_His', time()).'_create_websockets_statistics_entries_table.php'),
+                ], 'migrations');
+            }
 
-        $this
+            $this
             ->registerRoutes()
             ->registerDashboardGate();
 
-        $this->loadViewsFrom(__DIR__.'/../resources/views/', 'websockets');
-
-        $this->commands([
-            Console\StartWebSocketServer::class,
-            Console\CleanStatistics::class,
-            Console\RestartWebSocketServer::class,
-        ]);
+            $this->loadViewsFrom(__DIR__.'/../resources/views/', 'websockets');
+
+            $this->commands([
+                Console\StartWebSocketServer::class,
+                Console\CleanStatistics::class,
+                Console\RestartWebSocketServer::class,
+            ]);
+        } catch (QueryException $e) {
```

Comment:
If no Schema is used anymore, the exception should be gone.

Reply:
Done